extension/resources/templates/report.pug

```diff
@@ -27,10 +27,11 @@ html(lang="en")
                         .progress-bar.bg-success(role='progressbar', style='width:25%', aria-valuenow='15', aria-valuemin='0', aria-valuemax='100') #{passCount} Passed
```

Comment:
Can we hide the `All, Failed, passed` options when we are showing a single case result?

Reply:
sure, but for class level, even though it might only contains one method, still think it is necessary to show this as it is kind of a container